lib/registrant.rb

```diff
@@ -0,0 +1,28 @@
+class Registrant
+  attr_reader :name, :age, :permit, :earn_permit 
```

Comment:
`earn_permit` is a method, not an instance variable/attribute, so it doesn't need an `attr_reader`.